frontend/src/components/comments/CommentCard.jsx

```diff
@@ -24,19 +29,20 @@ export default function CommentCard({
   // Constants
   const commentCreatorName = comment.relatedCommentUser.name;
   const commentCreatorEmail = comment.relatedCommentUser.email;
-  const listOfCommentLikedUsers = comment.listOfCommentLikes.map(
-    (commentLike) => commentLike.likedCommentUser
-  );
 
   // Methods
   const handleDelete = () => {
     // Invoke the passed in event callback
     onDeleteClick(postId, comment);
   };
 
-  async function updateComment(updatedComment) {
+  async function updateComment(contentText) {
     try {
-      await CommentsApi.updateComment(postId, comment.id, updatedComment);
+      let formData = new FormData();
+      console.log(contentText);
+      formData.append("text", contentText);
+      console.log(formData);
```

Comment:
Console.logs could be removed unless you think it's handy to leave in for debugging related issues.